index.html

```diff
@@ -450,7 +450,7 @@
       </section>
       <section>
         <h3>
-          Presenting context: monitor available session(s) and say hello.
+          Receiving context: monitor available session(s) and say hello.
```

Comment:
At this point I don't believe "Receiving context" has been defined yet in the prose?